packages/mutation-testing-elements/src/components/mutation-test-report-totals.ts

```diff
@@ -0,0 +1,135 @@
+import { LitElement, html, property, customElement, css } from 'lit-element';
+import { bootstrap } from '../style';
+import { ResultTable, TableRow } from '../model/ResultTable';
+
+@customElement('mutation-test-report-totals')
+export class MutationTestReportTotalsComponent extends LitElement {
+
+  @property()
+  public model!: ResultTable;
+
+  public static styles = [bootstrap,
+    css`
+    th.rotate {
+      /* Something you can count on */
+      height: 50px;
+      white-space: nowrap;
+      padding-bottom: 10px;
+    }
+
+    th.rotate > div {
+      transform:
+      translate(27px, 0px)
+      rotate(325deg);
+      width: 30px;
+    }
+
+    .table-no-top>thead>tr>th {
+      border-width: 0;
+    }
+
+    .table-no-top {
+      border-width: 0;
+    }
+  `];
+
+  public render() {
+    return html`
+          <table class="table table-sm table-hover table-bordered table-no-top">
+            ${this.renderHead()}
+            ${this.renderBody()}
+          </table>
+      `;
+  }
+
+  private renderHead() {
+    return html`<thead>
+  <tr>
+    <th style="width: 20%">
+      <div><span>File / Directory</span></div>
+    </th>
+    <th colspan="2">
+      <div><span>Mutation score</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Killed</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Survived</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Timeout</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># No coverage</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Runtime errors</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Transpile errors</span></div>
```

Comment:
The property is called CompileError in the schema

Reply:
Thanks! I will change that.